packages/app/src/runner/index.ts

```diff
@@ -254,6 +260,8 @@ function runSpecE2E (spec: BaseSpec) {
   // create root for new AUT
   const $container = document.createElement('div')
 
+  $container.classList.add('screenshot-height-container')
```

Comment:
Is there any way to accomplish this same thing in a more declarative way? Eg, can we use a class in a template somehow? Ideally the screenshot logic should not leak into this part of the app, if possible - figuring out this class is added dynamically and then styled using `v-bind` is a lot of mental leaps someone will need to make in the future.

Reply:
Previously we had a style rule targeting `#unified-runner > div`, which actually led to the removal of a similar div from line 208 for a while because we couldn't tell the purpose. With the class we're being more clear about what's going on. Since we append the AUT to this wrapper div, I think we need to keep it here, but hopefully the class will make it clear in the DOM and code that this div is necessary for screenshots, so nobody "cleans it up".